run/GCHP/ExtData.rc.templates/ExtData.rc.TransportTracers

```diff
@@ -196,6 +196,17 @@ ZHANG_Rn222_EMIS  kg/m2/s N Y F%y4-%m2-01T00:00:00 none none rnemis   ./HcoDir/Z
 #==============================================================================
 TIMEZONES count N V - none none UTC_OFFSET ./HcoDir/TIMEZONES/v2015-02/timezones_voronoi_1x1.nc
 #
+#==============================================================================
+# --- O3 concentrations ---
+#==============================================================================
+GLOBAL_O3  1  N Y F%y4-%m2-01T00:00:00 none none SpeciesConc_O3 ./HcoDir/GCClassic_Output/13.0.0/$YYYY/GEOSChem.SpeciesConc.$YYYY$MM$DD_0000z.nc4
+#
+#==============================================================================
+# --- O3 loss rates ---
+#==============================================================================
+O3_LOSS  1  N Y F%y4-%m2-01T00:00:00 none none Loss_Ox ./HcoDir/GCClassic_Output/14.0.0/%y
```

Comment:
same question about 13.0.0 vs. 14.0.0 here

Reply:
The path has now been corrected in commit 59b64930.